pkg/move/atomic_test.go

```diff
@@ -0,0 +1,55 @@
+package move
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func mockCopyFunc(isSuccessful bool) copyFunc {
+	return func(fs afero.Afero) error {
+		if isSuccessful {
+			fs.MkdirAll("/work/tmp", 0755) //nolint:errcheck
```

Comment:
```suggestion
			_ = fs.MkdirAll("/work/tmp", 0755)
```

hardcoding the `"/work/tmp"` looks a bit error prone

Reply:
done https://github.com/Dynatrace/dynatrace-bootstrapper/pull/4/commits/a26ec38d7fa8e56ae0a5b200530ae2c4acd69a3a